boot/environ/lib/misc/rtlcompat.c

```diff
@@ -72,11 +72,21 @@ RtlAssert (
     IN PCHAR Message OPTIONAL
     )
 {
-    EfiPrintf(L"*** ASSERTION %s FAILED AT %d in %s (%s) ***\r\n",
-              FailedAssertion,
-              LineNumber,
-              FileName,
-              Message);
+    if (Message)
```

Comment:
What's the point of this?

Reply:
I suppose it's to avoid the message printed as "(null)" when Message == NULL.

Reply:
And why would Message be null?

Reply:
Because the prototype of the function let us suppose so (PCHAR message **OPTIONAL**)? (and that it's allowed by the "standard" RtlAssert() -- even if, here, we are talking about the RTL "compat" module of bootlib)

Reply:
As Hermès answered.
If we assume `Message != NULL`, then we should `ASSERT()` it, shouldn't we?

Reply:
Adding an `ASSERT()` instead.